x-pack/plugins/cases/docs/openapi/entrypoint.yaml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.0.1
+info:
+  title: Cases
+  description: OpenAPI schema for Cases endpoints
+  version: '0.1'
```

Comment:
What this version denotes? Kibana version? OpenAPI version? Cases endpoint version?

Reply:
Ok, make sense to me!